app/components/BasemapModal.js

```diff
@@ -24,27 +31,205 @@ const Button = styled.TouchableHighlight`
   elevation: 1;
 `
 
-export default class BasemapModal extends React.Component {
+// const CancelButton = styled.TouchableOpacity`
```

Comment:
Can we remove the commented out code before merging?

Reply:
Ah yes this is a question I have. Do we want the close button for the modal? I want to see how this works on the device and decide if we should also add a button. If the backdrop press works we can probably ignore the cancel button.

Reply:
@geohacker clicking outside the modal to close it seemed to work well for me and, to me, seemed quite intuitive.